tool/zjit_bisect.rb

```diff
@@ -58,40 +72,63 @@ def run_bisect(command, items)
   bisect_impl(command, [], items)
 end
 
+def run_ruby *cmd
+  stdout_data = nil
+  stderr_data = nil
+  status = nil
+  Open3.popen3(Shellwords.join cmd) do |stdin, stdout, stderr, wait_thr|
```

Comment:
IIRC, if you do it, it would shell out. It seems better to pass the array as is.

```suggestion
  Open3.popen3(*cmd) do |stdin, stdout, stderr, wait_thr|
```

Reply:
Not enough for capture3 :/ the process doesn't always die. I originally tried that but it will just wait until child exits

Reply:
You also don't need to *cmd, I think just passing `cmd` will just work.